Add support for sub-arg uniquness with :keys` (#314)
sorentwo/oban: README.md

* `:fields` — The fields to compare when evaluating uniqueness. The available
  fields are `:args`, `:queue` and `:worker`, by default all three are used.

* `:keys` — A specific subset of the `:args` to consider when comparing against
  historic jobs. This allows a job with multiple key/value pairs in the args to

>   be compared using only one or more of the key/values.

Reviewer: ```suggestion
  be compared using only a subset of them.
```
